src/components/calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
```

Comment:
[Optional] This is one of the project requirements but was missed by the previous reviewer so I highly encourage you to implement it.

- Please update your calculator component to be a functional component and not a class component.
<img width="284" alt="image" src="https://user-images.githubusercontent.com/37116322/236633887-4cd9da65-92e6-45db-ac1e-3e17e7d0b910.png">